code/game/gamemodes/wizard/artefact.dm

```diff
@@ -213,3 +213,383 @@
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+
+			if("assistant")
+				M.equip_to_slot_or_del(new /obj/item/clothing/under/color/grey(M), slot_w_uniform)
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(new /obj/item/clothing/shoes/black(M), slot_shoes)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+
+			if("animu")
+				M.equip_to_slot_or_del(new /obj/item/clothing/shoes/sandal(M), slot_shoes)
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(new /obj/item/clothing/head/kitty(M), slot_head)
+				M.equip_to_slot_or_del(new /obj/item/clothing/under/schoolgirl(M), slot_w_uniform)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+
+			if("cultist")
+				M.equip_to_slot_or_del(new /obj/item/clothing/head/culthood/alt(M), slot_head)
+				M.equip_to_slot_or_del(new /obj/item/clothing/suit/cultrobes/alt(M), slot_wear_suit)
+				M.equip_to_slot_or_del(new /obj/item/clothing/shoes/cult(M), slot_shoes)
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+
+			if("highlander")
+				M.equip_to_slot_or_del(new /obj/item/clothing/under/kilt(M), slot_w_uniform)
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(new /obj/item/clothing/head/beret(M), slot_head)
+				M.equip_to_slot_or_del(new /obj/item/clothing/shoes/combat(M), slot_shoes)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+
+			if("clown")
+				M.equip_to_slot_or_del(new /obj/item/clothing/under/rank/clown(M), slot_w_uniform)
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(new /obj/item/clothing/shoes/clown_shoes(M), slot_shoes)
+				M.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/clown_hat(M), slot_wear_mask)
+				M.equip_to_slot_or_del(new /obj/item/weapon/bikehorn(M), slot_l_store)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+
+			if("killer")
+				M.equip_to_slot_or_del(new /obj/item/clothing/under/overalls(M), slot_w_uniform)
+				M.equip_to_slot_or_del(new /obj/item/clothing/shoes/white(M), slot_shoes)
+				M.equip_to_slot_or_del(new /obj/item/clothing/gloves/color/latex(M), slot_gloves)
+				M.equip_to_slot_or_del(new /obj/item/clothing/mask/surgical(M), slot_wear_mask)
+				M.equip_to_slot_or_del(new /obj/item/clothing/head/welding(M), slot_head)
+				M.equip_to_slot_or_del(new /obj/item/device/radio/headset(M), slot_l_ear)
+				M.equip_to_slot_or_del(new /obj/item/clothing/suit/apron(M), slot_wear_suit)
+				M.equip_to_slot_or_del(new /obj/item/weapon/kitchen/utensil/knife(M), slot_l_store)
+				M.equip_to_slot_or_del(new /obj/item/weapon/scalpel(M), slot_r_store)
+				M.equip_to_slot_or_del(sword, slot_r_hand)
+				for(var/obj/item/carried_item in M.contents)
+					if(!istype(carried_item, /obj/item/weapon/implant))
+						carried_item.add_blood(M)
```

Comment:
Obviously this is a real proc, or it wouldn't have compiled; But this is -tg-'s standard for adding blood to an object, I question if it actually works for what you want to do, since we have such a massively different system.